tools/fetch-team-data.js

```diff
@@ -145,8 +145,10 @@ async function fetchBlogAuthors() {
         const contents = await fs.readFile(filename, "utf8");
         const { data: frontmatter } = matter(contents);
 
-        for (const username of frontmatter.authors) {
-            authors[username] = await fetchUserProfile(username);
+        if (Array.isArray(frontmatter.authors)) {
+            for (const username of frontmatter.authors) {
+                authors[username] = await fetchUserProfile(username);
+            }
```

Comment:
May be something like this? 
```suggestion
        const authors = frontmatter.authors;

        if (Array.isArray(authors)) {
            for (const username of authors) {
                authors[username] = await fetchUserProfile(username);
            }
```

Reply:
Same stuff. It's just stylistic personal preferences.

Reply:
Yep kinda, but when there might be a change in the value we might want to change at all occurrences. But we can avoid that by doing like this and change at one occurrence alone?